requirements/libraries.txt

```diff
@@ -8,11 +8,11 @@ django-csp==3.4
 django-extensions==2.1.3
 django-flags==4.2.4
 django-haystack==2.8.1
-# django-js-asset is required by teachers-digital-platform
+# django-js-asset is required by curriculum-review-tool
```

Comment:
Actually, these two requirements are required by the activity search tool, which we're still calling "teachers-digital-platform" in its new home in this repo.

```suggestion
# django-js-asset is required by teachers-digital-platform
```